en/simulation/gazebo.md

```diff
@@ -236,90 +257,72 @@ This thin wrapper simply embeds app arguments like current working directories a
 To start Gazebo and PX4 separately:
 
 * Run gazebo (or any other sim) server and client viewers via the terminal:
-  ```
+  ```sh
   make px4_sitl gazebo_none_ide
   ```
 * In your IDE select `px4_<mode>` target you want to debug (e.g. `px4_iris`)
 * Start the debug session directly from IDE
 
-This approach significantly reduces the debug cycle time because simulator (e.g. gazebo) is always running in background and you only re-run the px4 process which is very light.
+This approach significantly reduces the debug cycle time because simulator (e.g. Gazebo) is always running in background and you only re-run the px4 process which is very light.
 
 
-## Video Streaming
+## Video Streaming {#video}
 
 PX4 SITL for Gazebo supports UDP video streaming from a Gazebo camera sensor attached to a vehicle model.
-You can connect to this stream from *QGroundControl* (on UDP port 5600) and view video of the Gazebo environment from the simulated vehicle - just as you would from a real camera.
-The video is streamed using a *gstreamer* pipeline.
+When streaming is enabled, you can connect to this stream from *QGroundControl* (on UDP port 5600) and view video of the Gazebo environment from the simulated vehicle - just as you would from a real camera.
+The video is streamed using a *gstreamer* pipeline and can be enabled/disabled using a button in the Gazebo UI.
+
+The Gazebo camera sensor is supported/enabled on the following frames:
+* [Typhoon H480](#typhoon_h480)
```

Comment:
I believe there is now also the `iris_fpv_cam` which has the same camera.

Reply:
@julianoes I can't find `iris_fpv_cam` (or `gazebo_iris_fpv_cam`) or similar searching source tree, or issues or checking all make targets. I tried one ofthe `_vision` targets but that didn't work. 

Thoughts?

Reply:
I think @Jaeyoung-Lim is about to clean this up.

Reply:
Those targets were used by ROS, which doesn't require the corresponding world definitions. I will fix this in the world spawning PR :)

Reply:
Ah, so for now this is correct. Closing. Thanks.